packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -308,11 +308,17 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
                             // while the first caller is awaiting later async code in this block.
 
                             retrievedSnapshot = await cachedSnapshotP;
-
-                            method = retrievedSnapshot !== undefined ? "cache" : "network";
-
-                            if (retrievedSnapshot === undefined) {
-                                retrievedSnapshot = await this.fetchSnapshot(hostSnapshotOptions, scenarioName);
+                            if (retrievedSnapshot !== undefined) {
+                                method = "cache";
+                            } else {
+                                method = "network";
+                                const options: ISnapshotOptions = { ...hostSnapshotOptions };
+                                // Don't fetch the blobs/deltas if it is not the first call.
+                                if (!this.firstVersionCall) {
+                                    options.blobs = 0;
```

Comment:
Do we want to have options.deltas = 0? Or is this a default? I think we need to put a comment somewhere telling what are the defaults, as I have no clue :)